doc/releases/changelog-0.29.0.md

```diff
@@ -937,7 +940,7 @@
 * Only converts input to `qml.Hermitian` to a numpy array if the input is a list.
   [(#3820)](https://github.com/PennyLaneAI/pennylane/pull/3820)
 
-<h3>Contributors</h3>
+<h3>Contributors ✍</h3>
```

Comment:
I think here you can delete the returns and add a period to the end.

Reply:
It actually still renders nicely if we keep the names multiline:
![image](https://user-images.githubusercontent.com/49409390/221696721-56ff8265-0c96-45d4-be80-813f260a1281.png)